SnapML_models/requirements.txt

```diff
@@ -0,0 +1,3 @@
+timm
+onnx==1.11
+torch==1.11
```

Comment:
The `torch` & `timm` are included in the `requirements.txt` under the root dir. Maybe we can just modify the that file by including `onnx`.